### PR TITLE
Fix prometheus query logging/ticker bugs and allow passing nomad ca cart from env var

### PR DESCRIPTION
**Pull request by jvineet**
This is needed for us to run scalad in our nomad cluster.
Also changing the base image to alpine so we can exec into container and do some debugging.

Resolves:
https://github.com/zumper/zumper_code/issues/25084

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ WORKDIR /go/src/github.com/trivago/scalad
 RUN go get
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /scalad
 
-FROM scratch 
+FROM alpine:latest
 
 # Copy root filesystem
 COPY rootfs /
```
Comment (mgeggie) [inline Dockerfile]: Do we need this if we're running on Alpine?

File: main.go [hunk 2]
```diff
@@ -428,6 +432,7 @@ func main() {
 	log.Info("Port:            ", port)
 	log.Info("Nomad Host:      ", nomadHost)
 	log.Info("Nomad Region:    ", region)
+	log.Info("Nomad CA Cert:    ", nomadCaCert)
```
Comment (dbachelder) [inline main.go]: alignment is off

File: Dockerfile (hunk 1 above)
Comment (jvineet) [reply Dockerfile]: good point, we actually don't as the base alpine images ships with ca-certs. That was only for the scratch image where they were manuall adding ca-certs. I'll remove for now.

**Comment by dbachelder** [comment]
lgtm.. one format nit and questions about how to push back to upstream.

**Comment by jvineet** [comment]
A broke into 4 commits, the extra one being the changes I made to the debug level logging for prometheus queries, the earlier logs were misleading in terms of what query api call looked like.

**Comment by ebarriosjr** [comment]
Hi @jvineet it would be great if you could do a PR upstream.

**Pull request merged**